src/ui/components/AppWrapper/AppWrapper.tsx

```diff
@@ -151,92 +246,11 @@ const AppWrapper = (props: { children: ReactNode }) => {
       console.error(e);
       return;
     }
-
     dispatch(setPauseQueueIncomingRequest(true));
-    const connectionsDetails = await Agent.agent.connections.getConnections();
-    let userName: PreferencesStorageItem = { userName: "" };
-    const credentials = await Agent.agent.credentials.getCredentials();
-    const passcodeIsSet = await checkKeyStore(KeyStoreKeys.APP_PASSCODE);
-    const seedPhraseIsSet = await checkKeyStore(
-      KeyStoreKeys.IDENTITY_ROOT_XPRV_KEY
-    );
-    const passwordIsSet = await checkKeyStore(KeyStoreKeys.APP_OP_PASSWORD);
-    const storedIdentifiers = await Agent.agent.identifiers.getIdentifiers();
-
-    // @TODO - handle error
-    try {
-      const identifiersFavourites = await PreferencesStorage.get(
-        PreferencesKeys.APP_IDENTIFIERS_FAVOURITES
-      );
-      dispatch(
-        setFavouritesIdentifiersCache(
-          identifiersFavourites.favourites as FavouriteIdentifier[]
-        )
-      );
-    } catch (e) {
-      if (
-        !(e instanceof Error) ||
-        !(
-          e instanceof Error &&
-          e.message ===
-            `${PreferencesStorage.KEY_NOT_FOUND} ${PreferencesKeys.APP_IDENTIFIERS_FAVOURITES}`
-        )
-      ) {
-        throw e;
-      }
-    }
-
-    try {
-      const credsFavourites = await PreferencesStorage.get(
-        PreferencesKeys.APP_CREDS_FAVOURITES
-      );
-      dispatch(
-        setFavouritesCredsCache(
-          credsFavourites.favourites as FavouriteIdentifier[]
-        )
-      );
-    } catch (e) {
-      if (
-        !(e instanceof Error) ||
-        !(
-          e instanceof Error &&
-          e.message ===
-            `${PreferencesStorage.KEY_NOT_FOUND} ${PreferencesKeys.APP_CREDS_FAVOURITES}`
-        )
-      ) {
-        throw e;
-      }
-    }
-
-    try {
-      userName = await PreferencesStorage.get(PreferencesKeys.APP_USER_NAME);
-    } catch (e) {
-      if (
-        !(e instanceof Error) ||
-        !(
-          e instanceof Error &&
-          e.message ===
-            `${PreferencesStorage.KEY_NOT_FOUND} ${PreferencesKeys.APP_USER_NAME}`
-        )
-      ) {
-        throw e;
-      }
-    }
 
-    dispatch(
-      setAuthentication({
-        ...authentication,
-        userName: userName.userName as string,
-        passcodeIsSet,
-        seedPhraseIsSet,
-        passwordIsSet,
-        passwordIsSkipped: !passwordIsSet,
-      })
-    );
-
-    dispatch(setIdentifiersCache(storedIdentifiers));
-    dispatch(setCredsCache(credentials));
-    dispatch(setConnectionsCache(connectionsDetails));
+    await loadDatabase().catch((e) => {
+      /* TODO: handle error */
+    });
```

Comment:
We can't use the app without loading the DB so I would leave it throw.

And then when we add error handling we can do something more global to say there's an issue in case a number of different things won't work. Otherwise the risk is we bury this error.